src/routes/local.ts

```diff
@@ -1,10 +1,11 @@
-const express = require('express')
-const router = express.Router()
-const n2yo = require('./../api/n2yo')
-
-router.get('/two-line-elements/:id', (req, res) => {
-    n2yo.getTwoLineElements(req.params.id)
-        .then(data => res.json(data))
-})
-
-module.exports = router
\ No newline at end of file
+import express from 'express'
+import n2yo from './../api/n2yo/index.js'
+
+const router = express.Router()
+
+router.get('/two-line-elements/:id', (req, res) => {
+    n2yo.getTwoLineElements(req.params.id)
+        .then(data => res.json(data))
```

Comment:
`getTwoLineElements` already sends the response. It will be probably better to remove it from inside `getTwoLineElements`